CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ project (DLT-Viewer)
 
 set(CMAKE_CXX_STANDARD 11)
 
+set(CMAKE_PREFIX_PATH $ENV{QTDIR})
```

Comment:
Shouldn't this be `set(CMAKE_PREFIX_PATH "${CMAKE_PREFIX_PATH} $ENV{QTDIR}")` instead, to avoid overwriting existing prefix path?

Reply:
Yes, that would be safer. I will change it